ingine/ingine/Additional View Controllers/StatusViewController.swift

```diff
@@ -148,23 +150,23 @@ extension StatusViewController {
     }
     
     @objc func onLoadingProgressUpdate(_ notification: Notification) {
-        guard let dict = notification.userInfo,
-            let message = dict[NotificationProgressUserInfoType.Message.rawValue] as? String,
-            let startIndex = dict[NotificationProgressUserInfoType.StartingIndex.rawValue] as? Int,
-            let endIndex = dict[NotificationProgressUserInfoType.EndingIndex.rawValue] as? Int else {
-                return
-        }
-    
-        showMessage("\(message)\n\(Double((Double(startIndex)/Double(endIndex))*100).rounded(toPlaces: 0))% Complete", autoHide: false)
+//        guard let dict = notification.userInfo,
+//            let message = dict[NotificationProgressUserInfoType.Message.rawValue] as? String,
+//            let startIndex = dict[NotificationProgressUserInfoType.StartingIndex.rawValue] as? Int,
+//            let endIndex = dict[NotificationProgressUserInfoType.EndingIndex.rawValue] as? Int else {
+//                return
+//        }
+//
+//        showMessage("\(message)\n\(Double((Double(startIndex)/Double(endIndex))*100).rounded(toPlaces: 0))% Complete", autoHide: false)
```

Comment:
Why is this commented out

Reply:
I have created another class i.e. Notification Data Binging for this tasks that's why I commented this lines